.github/workflows/integration-build-product.yml

```diff
@@ -774,8 +798,424 @@ jobs:
+        if: ${{ inputs.with_simulation_tests }}
+        shell: bash
+        run: ssh robot "docker login ${{ env.REGISTRY }} -u ${{ secrets.registry_user }} -p ${{ secrets.registry_password }}"
+
+      - name: Remote - Install requirements
+        if: ${{ inputs.with_simulation_tests }}
+        shell: bash
+        run: ssh robot "sudo apt install python3-pip -y"
+
+      - name: Remote - Docker load spawner image
+        if: ${{ inputs.with_simulation_tests }}
+        shell: bash
+        run: |
+          ssh robot '
+            set -e
+            docker pull "${{ env.REGISTRY }}/ci/${{ inputs.product_name }}-noetic:${{ needs.Build-Spawner.outputs.raised_version }}"
+            docker tag "${{ env.REGISTRY }}/ci/${{ inputs.product_name }}-noetic:${{ needs.Build-Spawner.outputs.raised_version }}" "${{ env.REGISTRY }}/qa/${{ inputs.product_name }}-noetic:${{ needs.Build-Spawner.outputs.raised_version }}"
+          '
+
+      - name: Remote - Docker load simulator image
+        if: ${{ inputs.with_simulation_tests }}
+        shell: bash
+        run: |
+          ssh robot '
+            promoted_name=$(echo "${{ needs.Build-Simulator.outputs.image_name }}" | sed "s-/ci/-/qa/-g")
+            docker pull "${{ needs.Build-Simulator.outputs.image_name }}"
+            docker tag "${{ needs.Build-Simulator.outputs.image_name }}" $promoted_name
+          '
+
+      - name: Remote - Install CI Scripts
+        if: ${{ inputs.with_simulation_tests }}
+        shell: bash
+        run: ssh robot "python3 -m pip install integration-pipeline==$CI_INTEGRATION_SCRIPTS_VERSION --ignore-installed"
+
+      - name: Remote - unstash manifest
+        if: ${{ inputs.with_simulation_tests }}
+        shell: bash
+        run: scp product-manifest.yaml robot:${{ env.REMOTE_WORKSPACE }}/product-manifest.yaml
+
+      - name: Remote - unstash sim_configs
+        if: ${{ inputs.with_simulation_tests }}
+        shell: bash
+        run: scp -r simulator_artifacts robot:${{ env.REMOTE_WORKSPACE }}/simulator_artifacts
+
+      - name: Remote - unstash robot_jsons_noetic
+        if: ${{ inputs.with_simulation_tests }}
+        shell: bash
+        run: scp -r *.json* robot:${{ env.REMOTE_WORKSPACE }}/
+
+      - name: Remote - Setup QA Simulation tests
+        if: ${{ inputs.with_simulation_tests }}
+        shell: bash
+        id: simulator_tests_setup
+        run: |
+          ssh robot '
+            set -e
+            cd ${{ env.REMOTE_WORKSPACE }}
+
+            qa_key=simulator_tests
+
+            rm -f /tmp/target_dir.txt /tmp/version.txt /tmp/repo_name.txt /tmp/test_set.txt
+            export PATH="$HOME/.local/bin:$PATH"
+  
+            integration-pipeline get_yml_value --file product-manifest.yaml --key product_components.qa.$qa_key.target_dir --output_file /tmp/target_dir.txt
+            integration-pipeline get_yml_value --file product-manifest.yaml --key product_components.qa.$qa_key.version --output_file /tmp/version.txt
```

Comment:
The tests version is coming from a key on the product-manifest right? The way we have the [PR](https://github.com/MOV-AI/movai_tugbot/pull/419) it could come from the section for movai_tugbot, but maybe we can change this is the future if really needed

Reply:
i prefer the manifest. Standerdizing like the platform tests